### PR TITLE
[RPS-253] Add accessibility help page

### PR DESCRIPTION
**Pull request by ngta1**
Enabled a populated version of accessibility help document
to be bootstraped in the development module, with a blank
document being bootstraped in other environments on first
deployment for the user to populate and maintain (similar
to taxonomy and terms and conditions). For subsequent
deployments, as the document will already exist, the user
maintained version will not be overwritten.

Moved About document type from publicationsystem namespace
to common namespace.  Change involved updating config to
allow document types under the common namespace to also
be included as part of the resource location filters, which
enable automatic scanning for content-bean annotated classes.

### CODE REVIEW

File: repository-data/webfiles/src/main/resources/site/freemarker/common/base-footer.ftl [hunk 1]
```diff
@@ -14,9 +14,10 @@
             <div class="layout__item layout-2-3">
                 <ul class="footer__inner__list">
                     <@hst.link var="termslink" path="/publications/about/terms-and-conditions"/>
+                    <@hst.link var="accessibilitylink" path="/publications/about/accessibility-help"/>
                     <li class="footer__inner__list__item" title="Terms and Conditions"><a href="${termslink}" class="footer__inner__list__item__link">Terms and Conditions</a></li><!--
-                    --><li class="footer__inner__list__item"><a href="#" class="footer__inner__list__item__link">Privacy and cookies</a></li><!--
-                    --><li class="footer__inner__list__item"><a href="#" class="footer__inner__list__item__link">Accessibility help</a></li>
+                    --><li class="footer__inner__list__item"><a href="#" class="footer__inner__list__item__link">Privacy and cookies</a></li>
+                    <li class="footer__inner__list__item" title="Accessibility help"><a href="${accessibilitylink}" class="footer__inner__list__item__link">Accessibility help</a></li>
```
Comment (wojtek-oledzki) [inline repository-data/webfiles/src/main/resources/site/freemarker/common/base-footer.ftl]: missing `<!-- -->`

**Comment by ngta1** [comment]
Expanded some of the footer tests to test a footer link from each page (rather than just home page)

File: acceptance-tests/src/test/resources/features/site/footer.feature [hunk 2]
```diff
@@ -6,41 +6,63 @@ Feature: Footer
...
+        Then I should see a page titled "Accessibility help"
+        And I should see the content "Our site aims to comply with the World Wide Web Consortium's (W3C's) Web Accessibility Guidelines ..."
+
 
-    # Tests for each of the pages on the site
+    # Tests for each of the pages on the site can see footer, and a footer link works
     Scenario: Publications Overview
         Given I navigate to the "publications overview" page
         Then I should see the footer
+        When I can click on the "Terms and conditions" link
```
Comment (jacsamell) [inline acceptance-tests/src/test/resources/features/site/footer.feature]: It will probably read better on these if you get rid of the 'can'

Comment (ngta1) [reply acceptance-tests/src/test/resources/features/site/footer.feature]: Yep, "can" now removed.

**Pull request merged**